assets/src/styles/blocks/core-overrides/_media-text.scss

```diff
@@ -1,12 +1,26 @@
+@mixin adapt-padding-inline-start($containerWidth) {
+  padding-inline-start: calc(100% - (#{$containerWidth} / 2) + 0.75rem);
+}
+
+@mixin adapt-padding-inline-end($containerWidth) {
+  padding-inline-end: calc(100% - (#{$containerWidth} / 2) + 0.75rem);
```

Comment:
I'm wondering if, because of the alignfull declaration, the content should stick to the same vertical line as our logo, instead of giving it a little `+ 0.75rem` gap mirroring the content padding. Any thought on this ? I'm thinking about this because the [page header design](https://zeroheight.com/05f6e9516/v/latest/p/1285c9-page-header/b/88c480) has its content aligned on the logo (and it's using the media/text block too, so similar situation)

Reply:
I'm not sure, that would need to be discussed with the design team 🤔 Mag is the one who asked for this one to follow our grid. I guess the difference is that the page header is at the top of the page (so right below the logo) whereas this block will in most cases be placed in the middle of the page, so it would be strange to have it aligned with the logo and not with the rest of the content?